docs/modules/ROOT/pages/workflow-definition-from-file.adoc

```diff
@@ -0,0 +1,116 @@
+= How to load Workflow Definitions from files
+include::./includes/attributes.adoc[]
+
+This guide shows you how to load and use Serverless Workflow definitions from YAML files in your Quarkus application, making them available as CDI beans for injection and execution.
```

Comment:
```suggestion
This guide shows you how to load and use CNCF Workflow definitions from YAML files in your Quarkus application, making them available as CDI beans for injection and execution.
```
Also, please add the link to the DSL Specification in `CNCF Workflow definitions`.